tests/test_executable.py

```diff
@@ -357,11 +362,8 @@ def test_ssort_empty_dir(ssort, tmp_path):
 def test_ssort_non_existent_file(ssort, tmp_path):
     path = tmp_path / "file.py"
 
-    expected_msgs = [
-        f"ERROR: {escape_path(path)} does not exist\n",
-        "1 file was not sortable\n",
-    ]
-    expected_status = 1
```

Comment:
I think that the old behaviour is preferable.  If files are explicitly named then they should be sorted.  If they can't be sorted then it should be reported as an error.